include/capo/pcm.hpp

```diff
@@ -16,6 +17,8 @@ struct SampleMeta {
 	std::uint8_t channels{};
 };
 
+enum class FileFormat { eWav, eMp3, eFlac, eCOUNT_ };
```

Comment:
I'd prefer having the first entry be `eUnknown`, would also eliminate the optional in the other file.